src/condor_io/condor_crypt.cpp

```diff
@@ -96,13 +104,19 @@ Condor_Crypto_State::~Condor_Crypto_State() {
 }
 
 void Condor_Crypto_State::reset() {
-    dprintf(D_SECURITY | D_VERBOSE, "CRYPTO: resetting m_ivec(len %i) and m_num\n", m_ivec_len);
+    dprintf(D_ALWAYS, "CRYPTO: protocol(%i) resetting.\n", m_keyInfo.getProtocol());
```

Comment:
Decrease verbosity level - this gets invoked on every message for non-AES-GCM cases.